### PR TITLE
1. support International

### PR DESCRIPTION
**Pull request by pch957**
1. support International
2. add chinese language

### CODE REVIEW

**Comment by pch957** [comment]
depend on command: lupdate/lrelease

aptitude install qttools5-dev-tools

File: translations/keyhotee_zh_CN.ts [hunk 1]
```diff
@@ -0,0 +1,1222 @@
...
+    <message>
+        <location filename="../AddressBook/ContactView.ui" line="176"/>
+        <location filename="../AddressBook/ContactView.ui" line="256"/>
+        <source>Keyhotee ID</source>
+        <translation type="unfinished">Keyhotee ID</translation>
+    </message>
+    <message>
+        <location filename="../AddressBook/ContactView.ui" line="196"/>
+        <source>First Name</source>
+        <translation type="unfinished">姓</translation>
```
Comment (hackfisher) [inline translations/keyhotee_zh_CN.ts:103]: For Chinese, should "First name" be translated to surname(姓) ，or be translated to 名?
For west country, "Last name" is the surname, but the order is different for Chinese.

I don't know the convension to translate this, always make me confuse when fill the fields. But I think 名 for first name is better, cause I remember most sites use this way.

Comment (pch957) [inline translations/keyhotee_zh_CN.ts:103]: My english is very poor....
Appreciated if you can finish the translation

Comment (hackfisher) [inline translations/keyhotee_zh_CN.ts:103]: I'm not professional too, I would like to draft the rest translation after request is accepted. After that, I think maybe we could request proforssional writer to review it.

**Pull request merged**